src/pages/storage/storage-list.vue

```diff
@@ -0,0 +1,133 @@
+<template>
+  <Page title="Storage Settings" :breadcrumb-links="[]">
```

Comment:
Map tje errors to the "Page" component so, it'll be managed accordingly.
Eg:- https://github.com/SciGaP/emcenter-gateway/blob/master/src/pages/group/group-new-page.vue#L2